edison/templates/policy.html

```diff
@@ -0,0 +1,139 @@
+{% extends 'layout.html' %}
+
+{% block body %}
+
+
+<div class="policy">
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/Policy.css') }}">
+
+  <div class="list-policies" id="Policieslist">
+    <div class="cantainer">
+
+
+      <!-- Button trigger modal -->
+      <button class="btn btn-primary collapsed" data-toggle="collapse" data-target="#collapseAddPolicy" type="button"
+        aria-expanded="false" aria-controls="collapseAddPolicy">
+        Add Policy
+      </button>
+      <div class="collapse" id="collapseAddPolicy" style="width: 50rem;">
+        <div class="card card-body">
+          <form>
+            <div class="form-group">
+              <label for="PolicyName">Name</label>
+              <input type="text" class="form-control" id="PolicyName" placeholder="my policy">
+            </div>
+            <div class="form-group">
+              <label for="choiseRoom">Room</label>
+              <select class="form-control" id="choiceRoom">
+                <option> </option>
+                <option>Living Room</option>
+                <option>Bad Room</option>
+                <option>kitchen</option>
+                <option>children room 1</option>
+                <option>children room 2</option>
+              </select>
+            </div>
+            <div class="form-row">
+              <div class="form-group col-md-4">
+                <label for="light">Light</label>
+                <select class="form-control" id="light">
+                  <option> </option>
+                  <option>
+                    On </option>
+                  <option>Off</option>
+                </select>
+              </div>
+              <div class="form-group col-md-4">
+                <label for="ac">AC</label>
+                <select class="form-control" id="ac">
+                  <option> </option>
+                  <option>
+                    On </option>
+                  <option>Off</option>
+                </select>
+              </div>
+              <div class="form-group col-md-4">
+                <label for="shutters">Shutters</label>
+                <select class="form-control" id='shutters'>
+                  <option> </option>
```

Comment:
I think that in ac & light & shutters this option most stay because from my understating the user can add policy that change the light and the shutters , but don't do anything do the ac

Reply:
maybe we can write "stay the same " or something like this

Reply:
What if there isn't an AC in the room?
This is just like what I said in other conversation of ours.
Why not make the list of options dynamic and let the user choose what is inside a room.
When I'm saying let him choose I mean - create a list of options the user can choose from (AC, Lights, ...), and only then let him configure his plan base on what is actually in the room.

Reply:
I think you are right but this take a lot of time and this is not what I and the policy issue decided, so maybe we do it later in the project ?

Reply:
Ok so please remember to open a new issue about it :)